config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.sparql

```diff
@@ -0,0 +1,3 @@
+# Adding missing betrokken besturen kerkfabriek St.-Pieter van Puurs-Sint-Amands
```

Comment:
This one should be a `.graph` file no ?

Reply:
Indeed, I don't know why I was still convinced to put .sparql instead, I'll change that in other PRs as well. :man_facepalming: